src/torrent/utils/thread_interrupt.h

```diff
@@ -37,6 +37,8 @@
 #ifndef LIBTORRENT_UTILS_THREAD_INTERRUPT_H
 #define LIBTORRENT_UTILS_THREAD_INTERRUPT_H
 
+#include "config.h"
```

Comment:
This should never be in a header file.

Reply:
I have no quarrel with that, but it throws off clang-tidy since it can't infer what `lt_cacheline_aligned` is.